lizard_downloader.py

```diff
@@ -216,7 +217,7 @@ def show_data(self):
         """ Show the data as a new layer on the map """
 
         # Setup
-        WGS84 = "EPSG:4326"
+        wgs84 = "EPSG:4326"
```

Comment:
My bad, my version of flake8 did not check for constants. 
I'll change WGS84 back to upper case.